src/scroll-animation.js

```diff
@@ -49,6 +48,9 @@ export default class ScrollAnimation extends Component {
         clearTimeout(tid);
       })
     }
+    if (this.props.animateOnce && this.state.lastVisibility.completely) {
+      return;
```

Comment:
Hey @chloerice @nathanyoung , you two forked my fork before the changes in this commit were added to fix a bug with `animateOnce`. You'll probably want to port these changes into your own code. :)